wled00/pin_manager.cpp

```diff
@@ -214,7 +214,19 @@ bool PinManager::isPinOk(byte gpio, bool output)
     // JTAG: GPIO39-42 are usually used for inline debugging
     // GPIO46 is input only and pulled down
   #else
-    if (gpio > 5 && gpio < 12) return false;      //SPI flash pins
+
+    if ((strncmp_P(PSTR("ESP32-U4WDH"), ESP.getChipModel(), 11) == 0) ||    // this is the correct identifier, but....
+        (strncmp_P(PSTR("ESP32-PICO-D2"), ESP.getChipModel(), 13) == 0)) {  // https://github.com/espressif/arduino-esp32/issues/10683
+      // this chip has 4 MB of internal Flash and different packaging, so available pins are different!
+      if ((gpio == 1) || (gpio == 3) || ((gpio >= 6) && (gpio <= 8)) ||
```

Comment:
Hi @ladyada, thanks for your contribution 👍 

Is it necessary to also mark 1 and 3 (RX and TX) as unusable?

If I read the datasheet correctly, it looks like the board basicially has 16 and 17 connected to spi flash, while 9 and 10 are still availeable for use. 

There is a note in the datasheet rev 1.4 saying "GPIO6, GPIO7, GPIO8, GPIO11, GPIO16, and GPIO17 on the ESP32-U4WDH chip are connected to the SPI flash integrated in the chip and cannot be used for other purposes except for connecting external
PSRAM."

Reply:
i can adjust the statement to just reject 16/17, allow 9/10 and let the defaults determine the rest.

Reply:
Yes, sounds good. Please do.

Reply:
done!